examples/svm.py

```diff
@@ -110,3 +110,10 @@ def svm_from_cfg(cfg):
 inc_value = svm_from_cfg(incumbent)
 
 print("Optimized Value: %.2f" % (inc_value))
+
+
+# We can also validate our results (though this makes a lot more sense with instances)
```

Comment:
I think it still makes a lot of sense for machine learning as a user normally has a separate test set. @mlindauer do you think it would make the example too complicated to add a separate test set?